src/chef_workstation.js

```diff
@@ -47,26 +47,49 @@ function syncGetRegistryValue(baseKey, key, type = "REG_SZ") {
 
 function getVersion() {
   let manifestPath = null;
-  if (isDev) {
-    return "development";
-  } else if (is.windows()) {
+  const manifest = null;
+  if (is.windows()) {
     manifestPath = syncGetRegistryValue(CWS_REG_NODE, CWS_REG_KEY_INSTALL_DIR) + "version-manifest.json";
   } else {
     manifestPath = "/opt/chef-workstation/version-manifest.json";
   }
-  const manifest = require(manifestPath);
+  try {
+    manifest = require(manifestPath);
+  } catch(err) {
+    return "development";
```

Comment:
not a change for this pr but i wonder about returning development because it makes testing update paths hard. Wonder if we should return 0.0.0 or something so all that works? or update the update logic to do something better.